packages/edit-site/src/components/list/added-by.js

```diff
@@ -56,128 +144,16 @@ function BaseAddedBy( { text, icon, imageUrl, isCustomized, templateType } ) {
 	);
 }
 
-function AddedByTheme( { slug, isCustomized, templateType } ) {
-	const theme = useSelect(
-		( select ) => select( coreStore ).getTheme( slug ),
-		[ slug ]
-	);
-
-	return (
-		<BaseAddedBy
-			icon={ themeIcon }
-			text={ theme?.name?.rendered || slug }
-			isCustomized={ isCustomized }
-			templateType={ templateType }
-		/>
-	);
-}
-
-function AddedByPlugin( { slug, isCustomized, templateType } ) {
-	const plugin = useSelect(
-		( select ) => select( coreStore ).getPlugin( slug ),
-		[ slug ]
-	);
-
-	return (
-		<BaseAddedBy
-			icon={ pluginIcon }
-			text={ plugin?.name || slug }
-			isCustomized={ isCustomized }
-			templateType={ templateType }
-		/>
-	);
-}
-
-function AddedByAuthor( { id, templateType } ) {
-	const user = useSelect(
-		( select ) => select( coreStore ).getUser( id ),
-		[ id ]
-	);
+export default function AddedBy( { template } ) {
+	const addedBy = useAddedBy( template );
 
 	return (
 		<BaseAddedBy
```

Comment:
Is it worth keeping `AddedBy` and `BaseAddedBy` as separate components, or could they be merged into one?

Reply:
Oh yeah, makes sense! Will merge them into one.